clientv3/lease.go

```diff
@@ -184,6 +190,40 @@ func (l *lessor) Close() error {
 	return nil
 }
 
+func (l *lessor) keepAliveCtxCloser(ctx context.Context, id lease.LeaseID) {
+	activeCtx := true
+	for activeCtx {
+		select {
+		case <-ctx.Done():
+			activeCtx = false
```

Comment:
why do we need this activeCtx var? It seems like we always break the for loop no matter what happens. (l.donec returns directly, ctx.Done breaks the loop via this var)